AgentDiscoveries-Frontend/app/src/components/profile.js

```diff
@@ -79,17 +82,17 @@ export default class Profile extends React.Component {
 
     getProfileSrc() {
```

Comment:
Can't you now use the API directly, rather than fetching and converting to a data URL?

`<img src="/v1/api/pictures/{userId}">`

(This might not be as appropriate if you need specialised error handling)